### PR TITLE
Add the possibility to prefix entites with device name

### PR DESCRIPTION
**Pull request by tekand**
It was removed in 4210d84df993e181825901122beb53bafede2c85.
The problem is that Home Assistant uses the name of the entity to generate it's `entity_id` and not the provided `unique_id`. This can result in identical entities in Home Assistant (if using auto discovery).

Or maybe an alternative to this would be to provide the value of `unique_id` in `object_id` as well. Reference: https://www.home-assistant.io/integrations/mqtt/#naming-of-mqtt-entities

Or maybe both? ;)

### CODE REVIEW

**Comment by yozik04** [comment]
Prefixing `name` with `device name` is wrong.
https://developers.home-assistant.io/docs/core/entity#entity-naming

I think we need to provide unique_id.

People were requesting a prefix to distinguish between alarm panels: https://github.com/ParadoxAlarmInterface/pai/issues/329
Your pull request can be easily modified to cover that =)

**Comment by tekand** [comment]
I use HA as well, and have seen #381, but didn't find anything like that in my HA logs. Maybe I missed something.
Intead of `unique_id` you meant `object_id`, right? `unique_id` is already provided.

Regarding #329, it would only mean to remove 'Paradox' from it and leave only the serial, right? Or you prefer to have this configurable?

**Comment by yozik04** [comment]
> I use HA as well, and have seen #381, but didn't find anything like that in my HA logs. Maybe I missed something. Instead of `unique_id` you meant `object_id`, right? `unique_id` is already provided.

Sorry for confusion. I thought we do not provide `unique_id`. Then we should be all set. I do not think we need an `object_id` in addition.

> BTW are you still active on gitter? I have a problem with the target_state, that is a bit harder/longer to explain. :)

Probably I am. Have not seen any notifications for a long time already.

File: paradox/interfaces/mqtt/entities/abstract_entity.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def serialize(self):
         return dict(
             availability_topic=self.availability_topic,
             device=self.device,
-            name=f'{self.entity_name}',
+            name=f'Paradox {self.device.serial_number} {self.entity_name}' if cfg.MQTT_PREFIX_DEVICE_NAME else f'{self.entity_name}',
```
Comment (yozik04) [inline paradox/interfaces/mqtt/entities/abstract_entity.py:43]: Let's better have
```python3
f'Paradox {cfg.ENTITY_NAME_PREFIX} {self.entity_name}' if cfg.ENTITY_NAME_PREFIX else f'{self.entity_name}',
```

**Comment by yozik04** [comment]
Usage:
`MQTT_HOMEASSISTANT_ENTITY_PREFIX = ''` - default

`MQTT_HOMEASSISTANT_ENTITY_PREFIX = 'Paradox {serial_number} '` - will prefix with "Paradox abcdef"

Note the space in the end of prefix.

**Comment by clau-bucur** [comment]
Can this be documented or included in `pai.conf.example`?

**Pull request merged**